src/plugins/intel_cpu/src/node.cpp

```diff
@@ -834,13 +833,12 @@ void Node::prepareMemory(const DnnlMemoryDescPtr& intDesc, size_t indx) {
     auto weightCache = context->getWeightsCache();
     if (weightCache != nullptr && memory::format_kind::blocked == intDesc->getDnnlDesc().get_format_kind()) {
         const auto& format = intDesc->serializeFormat();
-        const uint64_t data_hash = weightCache->GetHashFunc().hash(
-                internalBlob->buffer(), internalBlob->byteSize());
+        const uint64_t data_hash =
+            weightCache->GetHashFunc().hash(static_cast<const unsigned char*>(internalBlob->getData()),
+                                            internalBlob->getSize());
 
-        const std::string string_hash = name + "_" + std::to_string(indx)
-                                        + "_" + format
-                                        + "_" + std::to_string(internalBlob->byteSize())
-                                        + "_" + std::to_string(data_hash);
+        const std::string string_hash = name + "_" + std::to_string(indx) + "_" + format + "_" +
+                                        std::to_string(internalBlob->getSize()) + "_" + std::to_string(data_hash);
```

Comment:
NIT. The previous format looked better (one line per term).

Reply:
Current is the result of clang-format, ok, restore to previous format.